kni.js

```diff
@@ -50,7 +50,7 @@ function main() {
 
         var states;
         if (config.fromJson) {
-            states = JSON.parse(knicript);
+            states = JSON.parse(kniscript); // FIXME this is broken
```

Comment:
This would be tested with the `-J` CLI flag and a single `*.json` argument, since there isn’t a sensible interpretation of multiple `*.json` arguments. This should probably be `kniscripts[0]` and an assertion that `kniscripts.length === 1`.

Reply:
I thing change this to `kniscripts[0]` and the FIXME is the test coverage gap.